scripts/ci/deploy_condition.sh

```diff
@@ -3,8 +3,16 @@
 set -e
 
 if [[ $TRAVIS_OS_NAME == "osx" && $TRAVIS_OSX_IMAGE != "xcode8.3" ]]; then
```

Comment:
Not sure about this osx logic - is it still required?

Reply:
@casperdcl Good catch! Not required anymore, it was a leftover from the older days.